mailpoet/lib/WooCommerce/Helper.php

```diff
@@ -114,4 +114,12 @@ public function wasMailPoetInstalledViaWooCommerceOnboardingWizard(): bool {
 
     return $installedViaWooCommerce;
   }
+
+  public function getOrdersTableName() {
+    if (!method_exists('\Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore', 'get_orders_table_name')) {
+      throw new \Exception('Cannot get orders table name when running a WooCommerce version that doesn\'t support custom order tables.');
```

Comment:
We have defined [a bunch of MailPoet exception classes](https://github.com/mailpoet/mailpoet/blob/1f34f57df5e330ee4baf844b013b419c26a9a662/mailpoet/lib/exceptions.php). We haven't replaced all usages of the default PHP exceptions but for new ones we should use our classes. Could you please repace the exception?

Reply:
Thanks, @costasovo. I agree that using a more specific and MailPoet exception is better. I'm thinking using `RuntimeException` is fine in this case as the more specific exceptions set a HTTP error code which doesn't make much sense in this context as far as I can see. What do you think?

Reply:
@rodrigoprimo, I think the Runtime exception is ok. It is a situation that should not happen.

Reply:
@costasovo, done. I added a new commit using `RuntimeException`. Could you please take a final look?